recipe/build.sh

```diff
@@ -84,6 +87,28 @@ if [[ $(uname) == "Linux" ]]; then
         REDUCE_RELOCATIONS=-reduce-relocations
     fi
 
+    # Remove private dependencies from select pkgconfig configuration files
+    # to help find gtk.
+    # This works around a bug in pkg-config
+    # https://github.com/conda-forge/conda-forge.github.io/issues/1880#issuecomment-1384518165
+    sed -i '/Requires.private:/d' "${CONDA_BUILD_SYSROOT}/usr/lib/pkgconfig/cairo.pc"
+    sed -i '/Requires.private:/d' "${CONDA_BUILD_SYSROOT}/usr/lib/pkgconfig/cairo-gobject.pc"
+    sed -i '/Requires.private:/d' "${CONDA_BUILD_SYSROOT}/usr/lib/pkgconfig/pango.pc"
+    sed -i '/Requires.private:/d' "${CONDA_BUILD_SYSROOT}/usr/lib/pkgconfig/gdk-pixbuf-2.0.pc"
+    sed -i '/Requires.private:/d' "${CONDA_BUILD_SYSROOT}/usr/lib/pkgconfig/gtk+-3.0.pc"
+    sed -i '/Requires.private:/d' "${CONDA_BUILD_SYSROOT}/usr/lib/pkgconfig/gdk-3.0.pc"
```

Comment:
@pkgw I'm not sure if you have issues with the approach taken here for CDTs,

Reply:
Well .... if we are going to use CDTs to provide these libraries, I think this is the sort of thing that is going to have to happen.